src/widgets/block.rs

```diff
@@ -121,9 +132,7 @@ pub struct Block<'a> {
 impl<'a> Default for Block<'a> {
     fn default() -> Block<'a> {
         Block {
-            title: None,
-            title_alignment: Alignment::Left,
-            title_on_bottom: false,
+            titles: None,
```

Comment:
Yes, but wouldn't it make more sense to rename them `titles_alignment` and `titles_on_bottom`. Furthermore, if we are expecting users to have that many titles to need functions that act on all their titles, shouldn't we instead provide a `.titles` function that would accept a vector of titles. We could also accept position and alignment parameters in that function so that it would act on the vector the just gave.

## Example
### Approach 1
```rust
Block::default()
  .titles(vec!["A", "B"])
  .titles_with_alignement(vec!["C", "D"], Alignment::Left)
  .titles_with_position(vec!["E", "F"), Position::Bottom)
  .titles_with_all(vec!["G", "H"], Alignment::Right, Position::Bottom)
  .title(Title::from("I").position(Position::Top).alignment(Alignment::Left));
```

### Approach 2

```rust
Block::default()
  .titles(vec!["A", "B"])
  .titles(Titles::from(vec!["C", "D"]).alignment(Alignment::Left))
  .titles(Titles::from(vec!["E", "F"]).position(Position::Bottom))
  .titles(Titles::from(vec!["C", "D"])
        .alignment(Alignment::Left)
        .position(Position::Bottom))
  .title(Title::from("I").position(Position::Top).alignment(Alignment::Left));
```
Personally, I prefer approach 2.

Reply:
I think keeping something that works fine over making a small breaking change is probably the right choice here. I don't think the ergonomics of having both `title` and `titles` seems easy.

I'm looking at this from the lens of an existing user and also from the perspective of maintaining a library long-term. Rename -> breaking a user. Adding a new function -> too many items to choose from when using completion in your editor or reading docs.

I think making title_alignment() work on all titles is fine. Linguistically it makes sense as a singular even though it acts on multiple elements ("The title alignment of the block is Left" or "The block's title alignment is center"), and from there title_position flows naturally.

Reply:
Sure, all right. Then I can make Title's `alignment` and `position` property optional, then make it so `title_alignment` and `title_position` changes all the alignments / positions of all titles except those that have already been set right?

Reply:
I've just done it and added some documentation.